l10n_fi_banks/data/res_bank.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="0">
+
+        <!-- Finnish banks -->
+
+        <!-- TODO: Add main branch address information -->
+        <!-- TODO: Add branches? -->
+
+        <!-- Disable the Reserve-account -->
+        <!-- We could also delete this, but there is always a chance
+          someone modified this to be an actual account -->
+        <record id="base.res_bank_1" model="res.bank">
+            <field name="active" eval="0" />
```

Comment:
I think it's best to leave this out. It's a bit hazardous to deactivate a record if this module isn't installed in a fresh Odoo installation.